lib/components/app/locale-selector.tsx

```diff
@@ -0,0 +1,98 @@
+import { CN, ES, FR, KR, US, VN } from 'country-flag-icons/react/3x2'
+import { connect, ConnectedProps } from 'react-redux'
+import { MenuItem, NavDropdown } from 'react-bootstrap'
+import { useIntl } from 'react-intl'
+import React, { MouseEvent } from 'react'
+import styled from 'styled-components'
+
+import * as uiActions from '../../actions/ui'
+import * as userActions from '../../actions/user'
+
+/**
+ * Renders flag icons for supported languages
+ * using emojis would be a simpler solution, but they are not available on Windows
+ * Exporting this in the config.js causes the a11y tests to fail. A better solution
+ * will need to be found if further customization is required.
+ */
+const FLAG_ICON_MAPPING: Record<string, React.ReactElement> = {
+  'en-US': <US style={{ width: 15 }} />,
+  'es-ES': <ES style={{ width: 15 }} />,
+  'fr-FR': <FR style={{ width: 15 }} />,
+  'ko-KR': <KR style={{ width: 15 }} />,
+  'vi-VN': <VN style={{ width: 15 }} />,
+  'zh-CN': <CN style={{ width: 15 }} />
+}
+
+const FlagContainer = styled.span`
+  &::after {
+    margin: 0 0.125em;
```

Comment:
This isn't working for me on safari. We might need `content: ''`?
<img width="87" alt="Screen Shot 2022-01-21 at 9 12 46 AM" src="https://user-images.githubusercontent.com/86619099/150541529-aa8ea664-ef64-4021-a569-9e816d1d7829.png">

Reply:
We can probably just throw the padding on the span itself right? do we need the pseudo element

Reply:
You have set the `content` attribute too, see for instance `SpanWithSpace`, etc.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/525/commits/46e67c33586931a4bf67960af05245df3520ff94

Reply:
Although lmk if you think just applying the styling on the span directly is preferable.

Reply:
Latest commit works great for me! I think I'd move away from the pseudo element but I'll leave that up to @binh-dam-ibigroup as the final reviewer